server/services/AI/AI.js

```diff
@@ -0,0 +1,48 @@
+const fs = require('fs');
+
+/* Random card picked*/
+exports.pickRandomCard = cards => {
+    const randomCard = cards[Math.floor(Math.random()*cards.length)];
+    return randomCard.cardId;
+}
+
+// Generate list of labels for each picture //
+async function autoWordGenerator() {
+    // Imports the Google Cloud client library
+    const vision = require('@google-cloud/vision');
+  
+    // Creates a client
+    const client = new vision.ImageAnnotatorClient();
+  
+    // Performs label detection on the image file
+    var cardId;
+    for (cardId=1; cardId < 248; cardId++) {
+        const [result] = await client.labelDetection('./src/images/cards/card (' + cardId + ').jpg');
+        const labels = result.labelAnnotations;
+        const descriptions = labels.map(label => label.description);
+        const data = JSON.stringify({cardId, descriptions});
+        fs.appendFileSync('card-labels.json', data + ", " + "\n");
```

Comment:
So are you writing to the file in one place and then reading from it again in another?